src/txmempool.h

```diff
@@ -534,6 +534,10 @@ class CTxMemPool
     const setEntries & GetMemPoolParents(txiter entry) const EXCLUSIVE_LOCKS_REQUIRED(cs);
     const setEntries & GetMemPoolChildren(txiter entry) const EXCLUSIVE_LOCKS_REQUIRED(cs);
     uint64_t CalculateDescendantMaximum(txiter entry) const EXCLUSIVE_LOCKS_REQUIRED(cs);
+
+    // track locally submitted transactions & periodically retry initial broadcast
```

Comment:
"periodically (REATTEMPT_BROADCAST_INTERVAL)"

Reply:
going to pass on this one, especially since I removed the constant for some variability in intervals.